docs/1-Applying/how_to_apply.md

```diff
@@ -4,16 +4,20 @@
 
 Some companies have a form you fill out and attach your resume to, and this is how you apply to their jobs. Sometimes this is the only way to apply to a company, because they want to keep all the applications together in their system.
 
-Often times, applying through this method lowers your liklihood of them getting back to you. Many many many people are applying for internships and it's a lot of work to get through all those resumes and select who to interview. Often many applications are just never looked at.
+However, this is not the best way to apply. From the company's perspective, application received online are lower quality on average and provide the least amount of information about the applicant. This means that they are less likely to get back to online applicants. Many many many people are applying for internships and it's a lot of work to get through all those resumes and select who to interview. Often many applications are just never looked at.
```

Comment:
> this is *often* not the best way...

I have seen companies that really try to level the playing field by getting everyone to apply online and not take external pushing as much as possible. This isn't super common though

also I'd add

> less likely to get back to online applicants *than applicants they learned about from other places (see below)*

just cause someone might be confused at that point in reading what 'other' is - I mean we get to it right away but it might make the read easier?